test/unit/model/QtiTestsDeleterTest.php

```diff
@@ -0,0 +1,336 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2023 (original work) Open Assessment Technologies SA.
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoMediaManager\test\unit\model;
+
+use core_kernel_classes_Class;
+use core_kernel_classes_Resource;
+use oat\generis\model\data\Ontology;
+use oat\tao\model\resources\Service\ClassDeleter;
+use oat\taoMediaManager\model\MediaService;
+use oat\taoMediaManager\model\QtiTestsDeleter;
+use oat\taoMediaManager\model\Specification\MediaClassSpecification;
+use oat\taoMediaManager\model\TaoMediaOntology;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Psr\Log\LoggerInterface;
+
+class QtiTestsDeleterTest extends TestCase
+{
+
+    private const MEDIA_URI = 'https://host/ontologies/tao.rdf#i123456789abcdef0123456789abcdef01';
```

Comment:
(personal opinion) No need to big strings here 

```suggestion
    private const MEDIA_URI = 'https://media#i123';
```